components/input/input.css

```diff
@@ -0,0 +1,54 @@
+.medium-label{
+    font-size: 2rem;
+}
+.primary-input{
+    padding: 1rem 1.5rem;
+    font-size: 2rem;
+    width: fit-content;
+    border-radius: 0.9rem;
```

Comment:
try using px for small scale bcoz border-radius is something we usually don't prefer to enlarge or minimize

Reply:
ok, I will do that thank you for the review